examples/catalog/lib/app_bar_bottom.dart

```diff
@@ -0,0 +1,129 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+/// Sample Catalog: AppBar with a custom bottom widget.
+///
+/// The bottom widget's TabPageSelector displays the relative position of the
+/// selected choice. The arrow buttons in the toolbar part of the app bar
+/// select the previous or the next choice.
+///
+/// Sample classes: [AppBar], [TabController], [TabPageSelector], [Scaffold], [TabBarView].
+///
+/// See also:
+///  * The "Components-Tabs" section of the material design specification:
+///    <https://material.io/guidelines/components/tabs.html>
+
+class Choice {
```

Comment:
Did we think about how to more clearly mark the code in the sample that is specific to the sample (like, for instance, this Choice class), and the code that is generic and meant to be copy-pasted by the developer? I found myself scanning through this code, looking for the UI-specific part (the part about the AppBar, TabController, etc). I'm immediately presented with this "app state" code, so I'm left wondering "do I need this?"

Not sure how to do it, but I'd love to clearly mark/separate/draw-the-eye to the generic parts of this sample. The part I'm going to copy and paste.